src/sentry/models/groupredirect.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import
+
+from sentry.db.models import BoundedBigIntegerField, Model
+
+
+class GroupRedirect(Model):
+    """
+    Maintains a reference from a group that has been merged (and subsequently
+    deleted) to the group that superceded it.
+    """
+    __core__ = False
+
+    group_id = BoundedBigIntegerField()
```

Comment:
I guess this should also be indexed since it will be used as filter criteria in the deletion task.